force-app/main/default/lwc/givingSummary/givingSummary.html

```diff
@@ -51,25 +54,25 @@ <h2 class="slds-text-heading_small slds-var-p-left_large">
               class="thisYear"
             >
             </lightning-formatted-number> </span
-          ><br />
-          <span>{labels.donationHistoryLabelThisYear}</span>
+          >
         </lightning-layout-item>
         <lightning-layout-item
-          size="12"
-          small-device-size="12"
+          size="4"
+          small-device-size="4"
           medium-device-size="4"
           large-device-size="4"
           class={lastElementClasses}
         >
-          <span class="previousYear bolder">
+          <span>{labels.labelPreviousYear}</span>
+          <br />
+          <span class="bolder">
             <lightning-formatted-number
               value={previousYear}
               format-style="currency"
               currency-code="USD"
             >
             </lightning-formatted-number> </span
-          ><br />
-          <span>{labels.donationHistoryLabelPreviousYear}</span>
+          >
```

Comment:
Can this formatting here get cleaned up a bit? Put the `</span>` on its own line with the bracket that got separated from it.

Reply:
Done!